scripts/install/kindr_install.bash

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -e  # exit on first error
+KINDR_VER="1.0.4"
+KINDR_URL="https://github.com/ethz-asl/kindr/archive/$KINDR_VER.zip"
+
+download_kindr() {
+    mkdir -p /usr/local/src/kindr
+    cd /usr/local/src/kindr
+    wget $KINDR_URL -O kindr-$KINDR_VER.zip
+    unzip kindr-$KINDR_VER.zip
+    cd -
+}
+
+install_kindr() {
+    # compile and install kindr
+    cd /usr/local/src/kindr
+    cd kindr-$KINDR_VER
+    mkdir -p build
+    cd build
+    cmake ..
+    sudo make install
```

Comment:
- `-j` option may be in future, but for a simple build we are really pre-mature optimizing here, this literally takes 3 seconds to build with default settings.
- I would agree, but in this case almost all the comands run in `/usr/local/` which requires sudo access, unless I'm wrong?